libplatsupport/mach_include/imx/platsupport/mach/serial.h

```diff
@@ -27,7 +28,7 @@ enum chardev_id {
     PS_SERIAL_DEFAULT = IMX_UART2
 
 #elif defined(CONFIG_PLAT_WANDQ) || defined(CONFIG_PLAT_NITROGEN6SX) \
-      || defined(CONFIG_PLAT_IMX7_SABRE) || defined(CONFIG_PLAT_IMX8MQ_EVK)
+      || defined(CONFIG_PLAT_IMX7_SABRE) || defined(CONFIG_PLAT_IMX8MQ)
```

Comment:
```suggestion
      || defined(CONFIG_PLAT_IMX7_SABRE) || defined(CONFIG_PLAT_IMX8MQ_EVK) \
      || defined(CONFIG_PLAT_MAAXBOARD)
```
Same here, this is really board specific and not a SOC or SOC-Family setting.